File: java/입문/삼각형의_완성조건_1.java
class Solution {
    public int solution(int[] sides) {
        int answer = 2;
        int maxSide = sides[0];
        int sumSide = 0;
        int idx = 0;
        for(int i =1; i < sides.length; i++) {
            if (maxSide < sides[i]) {
                maxSide = sides[i];
                idx = i;
            }
        }
        
        for(int i = 0; i < sides.length; i++) {
            if (idx == i) {
                continue;
            }
            sumSide += sides[i];
        }
        
        if (maxSide < sumSide) {
            answer = 1;
        }
        return answer;
    }
}

File: swift/입문/삼각형의_완성조건_1.swift
import Foundation

func solution(_ sides:[Int]) -> Int {
    var sides3 = sides
    sides3.sort()
    if sides3[0] + sides3[1] > sides3[2] {
        return 1
    } else {
        return 2
    }
}

// max는 옵셔널 반환